src/components/Sidebar/index.jsx

```diff
@@ -49,9 +49,10 @@ export default class Sidebar extends Component {
       CSS: <LanguageCss3Icon />,
     };
 
+    // Add languages that does not have BUGZILLA MAPPING
```

Comment:
I'm not sure I understand this comment. Can you elaborate? Also, when I said to add a comment, I meant explaining why we are doing `['Swift', ...Object.keys(BUGZILLA_LANGUAGES)]` instead of just including `Swift` in `BUGZILLA_LANGUAGES`.